src/backend/common/deferred/clients/gcloud_client.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional, Type
+
+from google.cloud import tasks_v2
+
+from backend.common.deferred.clients import TaskClient
+from backend.common.deferred.queues import GCloudTaskQueue
+from backend.common.deferred.requests import GCloudTaskRequest
+
+
+_DEFAULT_PROJECT = "test"
+_DEFAULT_LOCATION = "us-central1"
+
+
+class GCloudTaskClient(TaskClient[GCloudTaskQueue]):
+    def __init__(
+        self, *, request_type: Type[GCloudTaskRequest], base_url: Optional[str] = None
+    ) -> None:
+        self.request_type = request_type
+        self.base_url = base_url
+
+        super().__init__(tasks_v2.CloudTasksClient())
+
+    def queue(self, name) -> GCloudTaskQueue:
+        from backend.common.environment import Environment
+
+        gcloud_project = Environment.project()
```

Comment:
We might look to pass in the `project` as opposed to divining it from the Environment here